cli/jobs/deepspeed/deepspeed-autotuning/generate-yml.sh

```diff
@@ -2,13 +2,15 @@
 # Generate key
 ssh-keygen -t rsa -f './src/generated-key' -N ''
 
-# Generate yaml file with key path
+# Pre-set process_count_per_instance so it can be passed into deepspeed via bash script.
+process_count_per_instance=2
```

Comment:
not necessarily a big deal, but should this be renamed to number of GPUs per node?

Reply:
yeah I think that makes it clearer, I renamed it